plugins/modules/nd_site.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Anvitha Jain (@anvitha-jain) <anvjain@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: nd_site
+short_description: Manage sites on Nexus Dashboard.
+description:
+- Manage sites on Nexus Dashboard which are then used by Nexus Dashboard Orchestrator (NDO) >= 2.2(2d).
+author:
+- Anvitha Jain (@anvitha-jain)
+options:
+  site_password:
+    description:
+    - The password for the APIC.
+    type: str
+  site_username:
+    description:
+    - The username for the APIC.
+    type: str
+  login_domain:
+    description:
+    - The AAA login domain for the username for the APIC.
+    type: str
+  inband_epg:
+    description:
+    - The AAA login domain for the username for the APIC.
+    type: str
+  site_name:
+    description:
+    - The name of the site.
+    type: str
+    aliases: [ name, fabric_name ]
+  # ToDo: To avail the securityDomains feature, ND v4 API update is required.
```

Comment:
Was it discussed to leave these commented here and in other places of the module?